{"name": "Проверка неуспешного создания курьера - курьер с таким логином существует", "status": "passed", "start": 1730235300175, "stop": 1730235300343, "uuid": "b96acd8e-4f5b-4acb-9ebb-9caa497d121f", "historyId": "823d6c1b09227222a1febb002af304a3", "testCaseId": "823d6c1b09227222a1febb002af304a3", "fullName": "tests.test_create_courier.TestCreateCourier#test_cant_create_two_couriers_with_same_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "home"}, {"name": "thread", "value": "15252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}